pkgs/applications/version-management/git/update.sh

```diff
@@ -10,7 +10,7 @@ targetVersion="${1:-$latestTag}"
 if [ ! "${oldVersion}" = "${targetVersion}" ]; then
   update-source-version git "${targetVersion}"
   nixpkgs="$(git rev-parse --show-toplevel)"
-  default_nix="$nixpkgs/pkgs/applications/version-management/git-and-tools/git/default.nix"
+  default_nix="$nixpkgs/pkgs/applications/version-management/git/default.nix"
```

Comment:
Shouldn't that rather be relative to $0?

Reply:
I aimed to make the minimal change to get this working; I have no objection to making more extensive changes, but I'd like to defer to folk with more experience on this!